pre_commit/git.py

```diff
@@ -197,6 +197,11 @@ def commit(repo: str = '.') -> None:
     cmd_output_b(*cmd, cwd=repo, env=env)
```

Comment:
Some... interesting choices made in the surrounding code here. Agree it's a good idea to mirror those, but `'.'` as opposed to `None` and the `*cmd` instead of just putting the hard coded args there is... odd lol.